helix-core/src/test.rs

```diff
@@ -152,7 +152,7 @@ pub fn print(s: &str) -> (String, Selection) {
 pub fn plain<R: Into<Rope>>(s: R, selection: &Selection) -> String {
     let s = s.into();
     let primary = selection.primary_index();
-    let mut out = String::with_capacity(s.len_bytes() + 5 * selection.len());
+    let mut out = String::with_capacity(s.len_chars() + 5 * selection.len());
```

Comment:
This looks unrelated and also incorrect. Strings only oeprste in terms of byte indecies so you should reserve bytes (it would also be more efficent and easier to read if you just initialized out to `s.to_string()` and avoided the copy but this is test code so I would just remove that from this PR

Reply:
Oh I did not mean to check this in. Will remove.